src/baldr/graphreader.cc

```diff
@@ -23,50 +22,37 @@ constexpr size_t AVERAGE_MM_TILE_SIZE = 1024;         // 1k
 namespace valhalla {
 namespace baldr {
 
-struct GraphReader::tile_extract_t {
-  tile_extract_t(const boost::property_tree::ptree& pt) {
-    // if you really meant to load it
-    if (pt.get_optional<std::string>("tile_extract")) {
-      try {
-        // load the tar
-        archive.reset(new midgard::tar(pt.get<std::string>("tile_extract")));
-        // map files to graph ids
-        for (auto& c : archive->contents) {
-          try {
-            auto id = GraphTile::GetTileId(c.first);
-            tiles[id] = std::make_pair(const_cast<char*>(c.second.first), c.second.second);
-          } catch (...) {
-            // skip files we dont understand
-          }
+GraphReader::tile_extract_t::tile_extract_t(const boost::property_tree::ptree& pt) {
+  std::cout << "Initizliaing a tile extract" << std::endl;
```

Comment:
Change to LOG_INFO/DEBUG instead?

Reply:
It's just temporary while I track down all the usages - will be removed in the final version.  I should've marked this PR as a draft.